### PR TITLE
[FIX] web: adapt bold layout to new header

### PR DESCRIPTION
**Pull request by proose**
The previously created company_details field was introduced to allow customers to manually edit part of their pdf headers. Only the bold layout was not updated with this field.

### CODE REVIEW

File: addons/web/views/report_templates.xml [hunk 1]
```diff
@@ -384,17 +384,9 @@
...
-                            <span t-if="forced_vat" t-esc="forced_vat"/>
-                            <span t-else="" t-field="company.vat"/>
-                        </li>
-                        <li t-if="company.phone">Tel: <span class="o_force_ltr" t-field="company.phone"/></li>
-                        <li t-if="company.email"><span t-field="company.email"/></li>
-                        <li t-if="company.website"><span t-field="company.website"/></li>
-                    </ul>
+                    <div name="company_address">
+                        <span t-if="company.company_details" t-field="company.company_details"></span>
+                    </div>
```
Comment (vin-odoo) [inline addons/web/views/report_templates.xml:389]: Don't put a div with the name company_address alone inside a div named company_address too :smile: 
I think you can just remove the inner div completely

**Review by vin-odoo — CHANGES_REQUESTED** [review]

File: addons/web/views/report_templates.xml (hunk 1 above)
Comment (proose) [reply addons/web/views/report_templates.xml:389]: Oops, indeed

**Review by vin-odoo — COMMENTED** [review]
@proose This is not linked to any task etc?

**Comment by proose** [comment]
@vin-odoo it was originally linked to a feature we wanted, from the pad, which we then dropped. We found out this error instead. I didn't create a whole task for it as the PR is small and was already done

**Review by vin-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**